### PR TITLE
Python code to determine the sediment of sentence

### LINKED ISSUE #55: Create a Python algorithm to determine the sentiment of a sentence
Use NLTK to develop the algorithm

What is the sentiment of the sentence - positive, negative, neutral

Keep in mind that feedback can have multiple sentences.

### PR DESCRIPTION
**Pull request by eyerusalemdani**
Closes #55 

**Task Description**

This code is using nltk library for determining the sediment of sentences in to Positive, Neutral, Negative.

**Time Spent**

| **Activity** | **Time Spend** |
| ----------- | --------- |
|Research resources on how to able to sediment of data | 3 hours |
| Reading the Json file and convert to a list and define a sentiment analysis and then define a list finally plot in pie chart | 8 hours |

**How to access instruction**

- First download the floop-data-15k Json file from S3 
- Import the following libraries such as 

      1. import string
      2. import matplotlib.pyplot as plt
      3. from nltk.sentiment.vader import 
           SentimentIntensityAnalyzer.
      4. import matplotlib.pyplot as plt

- After running the code, Enter the file path of the **_floop-data-15k file_** path and press `enter`

- Finally, you will get the outcome of sediment of the data in the file which you have given to the code, these are Neutral, positive and negative with pie chart.

**To test**

**Enter the file path **
s-2-2

**After the dataset imported and listed **
S-2-3

**Pie chart**

s-2 -1

### CODE REVIEW

File: backend/nlp_sent_analysis.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    return list(new_ds)
+
+
+new_ds = clean_dataset(dataset)
+print('Cleaned dataset')
+
+
+
+def sentiment_analyse(sentiment_text):
+#     print('{}: '.format(sentiment_text), end = '')
```
Comment (dale108) [inline backend/nlp_sent_analysis.py:33]: I would remove this commented out code

Comment (eyerusalemdani) [reply backend/nlp_sent_analysis.py:33]: I removed it.

File: backend/nlp_sent_analysis.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import string
+
+import matplotlib.pyplot as plt
+from nltk.sentiment.vader import SentimentIntensityAnalyzer
+import matplotlib.pyplot as plt
+
+print('Libraries imported.')
```
Comment (dale108) [inline backend/nlp_sent_analysis.py:7]: generally we don't need to print anything after import statement successfully do what they do

**Comment by dale108** [comment]
I'm pretty sure that this is operating on terms, not sentences, but you've got all the pieces here, so as long as python can interpret the code, I'll accept

**Review by dale108 — APPROVED** [review]
See other comments for my overall concerns

**Review by Abenezer1041 — APPROVED** [review]
Looks good. It worked as it was designed

**Comment by eyerusalemdani** [comment]
I have improved the code running instruction according to the comment I get from class.

File: backend/nlp_sent_analysis.py (hunk 1 above)
Comment (TurkiBK) [reply backend/nlp_sent_analysis.py:33]: Looks good, but if you make variable name more clear .For example, dataTest and newDataTest.

**Review by TurkiBK — APPROVED** [review]
works.

**Pull request merged**